doc/05-Voters.md

```diff
@@ -0,0 +1,50 @@
+Vote for the current page
+=========================
+
+Basics
+------
+
+To guess what's the current page inside your menu, KnpMenu use a Voter mechanism.
+
+You have to define an `ItemMatcher` that use one or many voter to decide if the page is or not the current page.
```

Comment:
you don't define a matcher. You instantiate it. The class definition is already part of KnpMenu